react/testUtils/product-utils.js

```diff
@@ -0,0 +1,35 @@
+export const getProduct = () => ({
```

Comment:
It isn't better to put this file on the `__mock__`  folder?

Reply:
I don't think so. I just created an utils to generate Product. `__mock__` folder seems like a place to add files that mock some external lib (or something like that).

Reply:
but since it is a plain const variable, with no real logic envolved, I think it will be more suitable to add it on the mock folder.